src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -91,6 +95,55 @@ private enum Booleans : uint
             IsWebSocketRequest = 0x00000800,
             Default = AllowAutoRedirect | AllowWriteStreamBuffering | ExpectContinue
         }
+
+        private class HttpClientParameters
+        {
+            public readonly DecompressionMethods AutomaticDecompression;
+
+            public readonly bool AllowAutoRedirect;
+
+            public readonly int MaximumAutomaticRedirections;
+
+            public readonly int MaximumResponseHeadersLength;
+
+            public readonly bool PreAuthenticate;
+
+            public readonly int Timeout;
+
+            public readonly SslProtocols SslProtocols;
+
+            public readonly bool CheckCertificateRevocationList;
```

Comment:
Nit: please remove the blank lines between all of these fields.

Reply:
Done